pipeline/src/lib.rs

```diff
@@ -41,10 +48,26 @@ pub async fn session_from_config<F: ConclusionFeed + 'static>(
         .with_information_schema(true);
 
     let mut ctx = SessionContext::new_with_config(session_config);
-    ctx.register_table(
-        "conclusion_feed",
-        Arc::new(conclusion::FeedTable::new(config.conclusion_feed)),
-    )?;
+    match config.conclusion_feed {
+        ConclusionFeedSource::Direct(conclusion_feed) => {
+            ctx.register_table(
+                "conclusion_feed",
+                Arc::new(conclusion::FeedTable::new(conclusion_feed)),
+            )?;
+        }
+        #[cfg(test)]
+        ConclusionFeedSource::InMemory(table) => {
+            debug_assert_eq!(
```

Comment:
should this be a regular assert in case we run tests in release mode? probably fine as is just a thought.

Reply:
good call